[CLOUD-878] minor fixes and links from README (#1073)
percona/percona-docker: README.md

See individual directories for instructions on how to build and run Percona software.

> You can find images that we build in Docker Hub registry. We have two organizations:

Reviewer: ```suggestion
The images we build can be found in the Docker Hub registry. We have two organizations:
```
